cmd/factory-sync/main.go

```diff
@@ -0,0 +1,58 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
why not directly "sync" command?

Reply:
Just to make it clearer that it is only for deployment in a factory. In some ways this could be seen as a separate project.